Specs/Scene/ModelSpec.js

```diff
@@ -150,6 +152,12 @@ defineSuite([
         verifyRender(duckModel);
     });
 
+    it('resolves readyPromise', function() {
+        waitsForPromise(duckModel.readyPromise, function(model) {
+            verifyRender(model);
```

Comment:
I know it is already implicit, but perhaps we should explicitly validate `model.ready`.